runtime/nls/port/port.nls

```diff
@@ -495,3 +495,13 @@ J9NLS_PORT_DUMP_PATH_EXISTS.explanation=The JVM was unable to produce a system d
 J9NLS_PORT_DUMP_PATH_EXISTS.system_action=The JVM continues, the requested system dump is not produced.
 J9NLS_PORT_DUMP_PATH_EXISTS.user_response=Check whether the name of a non-regular file (for example a directory, symbolic link, pipe or socket) in the current working directory of the Java application clashes with the default system dump file name (core or core.<process ID>) or with the setting of /proc/sys/kernel/core_pattern and /proc/sys/kernel/core_uses_pid. Either rename the subdirectory, or change the settings of /proc/sys/kernel/core_pattern and /proc/sys/kernel/core_uses_pid so that the names do not clash.
 # END NON-TRANSLATABLE
+
+J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND=The core file created by child process with pid = %i was not found. Review the documentation for the %s program \"%s\" to find where the core file is written and ensure that program does not truncate it.\n
+# START NON-TRANSLATABLE
+J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND.sample_input_1=12345
+J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND.sample_input_2=/proc/sys/kernel/core_pattern
+J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND.sample_input_3=|/usr/lib/systemd/systemd-coredump %P %u %g %s %t %c %h
+J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND.explanation=Tools such as systemd-coredump use /proc/sys/kernel/core_pattern to process the core dump, for example to send it to a different directory and manage total core dump disk usage, or to suppress it.
+J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND.system_action=The VM could not find the core dump in the current directory most likely because the external program sent it elsewhere or suppressed it.
+J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND.user_response=Refer to the documentation for the external program specified in /proc/sys/kernel/core_pattern to find the core dump and review the program's configuration to ensure the dump is not truncated.
```

Comment:
omr depends on the definition of this message in openj9. For example, here is the usage of `J9NLS_PORT_LINUXDUMP_PIPE_CORE`: https://github.com/eclipse/omr/blob/ef4d57d/port/linux/omrosdump_helpers.c

So I presume that since openj9 depends on omr, that once J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND is fully merged, then omr can be updated to use this new message. I guess something must generate the following IDs:

```
$ grep -r J9NLS_PORT_LINUXDUMP_PIPE_CORE *
nls/portnls.h:#define J9NLS_PORT_LINUXDUMP_PIPE_CORE__MODULE 0x504f5254
nls/portnls.h:#define J9NLS_PORT_LINUXDUMP_PIPE_CORE__ID 30
nls/portnls.h:#define J9NLS_PORT_LINUXDUMP_PIPE_CORE J9NLS_PORT_LINUXDUMP_PIPE_CORE__MODULE, J9NLS_PORT_LINUXDUMP_PIPE_CORE__ID
port/linux/omrosdump_helpers.c:			portLibrary->nls_printf(portLibrary, J9NLS_WARNING | J9NLS_STDERR, J9NLS_PORT_LINUXDUMP_PIPE_CORE, corePatternFileName, corePatternFormat, pid);
```

Reply:
The java.properties file contains the following
```
PORT030=%s setting "%s" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.%d.\n
```

Once this is merged you can look there to find the new number (which looks like it will be 49).